### PR TITLE
server: tiny update `ConnTestSuite`

### PR DESCRIPTION
**Pull request by tiancaiamao**
### What problem does this PR solve? 

`ConnTestSuite` does not call `domain.Close` and `store.Close` to release resource in the test code.

### What is changed and how it works?

Tiny update `ConnTestSuite`... and if `domain.Close` is called, the `expensive.Handle.Close()` is not necessary.

### Check List 

Tests 

 - No code
This is the change of the unit test code itself.

### CODE REVIEW

**Comment by tiancaiamao** [comment]
PTAL @bb7133 @winkyao

File: util/expensivequery/expensivequery.go [hunk 1]
```diff
@@ -78,11 +78,6 @@ func (eqh *Handle) Run() {
 	}
 }
 
-// Close closes the handle and release the background goroutine.
```
Comment (crazycs520) [inline util/expensivequery/expensivequery.go]: Why remove this? 

em... Actually, this function is useless, but, It ok to not close  `exitCh `?

Comment (tiancaiamao) [reply util/expensivequery/expensivequery.go]: `extiCh` is closed in both `domain.Close` and `expensiveQueryHandl.Close`, lead to panic. @crazycs520

Comment (zz-jason) [reply util/expensivequery/expensivequery.go]: /cc @XuHuaiyu

**Pull request merged**